src/OrleansConsulUtils/ConsulBasedMembershipTable.cs

```diff
@@ -6,23 +6,21 @@
 using Orleans.Messaging;
```

Comment:
Please split to 1 class per file, so when separating the membership, gateway packages, no need to modify files, just move them between projects.